pkg/projects/resource_project.go

```diff
@@ -203,11 +240,20 @@ func projectResource() *schema.Resource {
 		}
 
 		data.SetId(key)
+
+		_, err = updateUsers(key, users, m)
+		if err != nil {
+			return diag.FromErr(err)
+		}
+
 		return readProject(ctx, data, m)
 	}
 
 	var updateProject = func(ctx context.Context, data *schema.ResourceData, m interface{}) diag.Diagnostics {
-		project, key, err := unpackProject(data)
+		log.Printf("[DEBUG] updateProject")
+		log.Printf("[TRACE] %+v\n", data)
+
+		key, project, users, err := unpackProject(data)
```

Comment:
Remember the lessons of artifactory: If you get a 404 (or in RT, 400), then set the `id` to `""` to indicate it's been deleted. You should do that here too